<?xml version="1.0" ?>
<launch>
    <arg name="output"        default="screen"/>

    <node pkg="drone_odometry" type="odom_cov_overwrite" name="odom_cov_overwrite" output="$(arg output)" >

      <param name="odom1_in_topic" type="string" value="/t265/odom/sample"/>  
      <param name="cov1_mult_pos" value="0.0001"/>  
      <param name="cov1_mult_vel" value="0.001"/>  

    </node>

</launch>
